mappings/net/minecraft/client/render/model/ModelRotationContainer.mapping

```diff
@@ -1,2 +1,3 @@
 CLASS dpe net/minecraft/client/render/model/ModelRotationContainer
 	METHOD b getRotation ()Ldoz;
+	METHOD c isUVLocked ()Z
```

Comment:
Not completely sure of the standard -- should this be `isUvLocked`?

Reply:
Yes.